libs/ktxreader/include/ktxreader/Ktx1Reader.h

```diff
@@ -51,7 +51,6 @@ namespace Ktx1Reader {
     PixelDataFormat toPixelDataFormat(const KtxInfo& info);
     bool isCompressed(const KtxInfo& info);
     TextureFormat toTextureFormat(const KtxInfo& info);
-    TextureFormat toSrgbTextureFormat(TextureFormat tex);
```

Comment:
The original implementation of this function contained code that would have to be updated now, but I couldn't find the implementation anymore. So I deleted the declaration. Let me know if I missed sth!